src/utils/file.ts

```diff
@@ -404,3 +405,12 @@ export async function readEnvironmentVariablesFile(
     }
     return envVars;
 }
+
+// Inform local processes to interrupt when a deployment has started
+export async function interruptLocalProcesses() {
+    const interruptLocalPath = path.join(os.homedir(), ".geneziointerrupt");
+    const interruptLocal = await fsPromise.open(interruptLocalPath, "w");
+    await interruptLocal.close();
```

Comment:
1. What happens if there is no such path?
2. Why open and close if you modify the access time below?